ExampleMod/Common/GlobalItems/ExampleExperienceItem.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Terraria;
+using Terraria.ModLoader;
+using Terraria.ID;
+using Microsoft.Xna.Framework;
+using Terraria.ModLoader.IO;
+
+namespace ExampleMod.Common.GlobalItems
+{
+	public class ExampleExperienceItem : GlobalItem {
```

Comment:
`ExampleMeleeExperienceGlobal`?

Reply:
Needs a comment about the mechanic... currently, spawns coins when upgraded? I dunno...